### DISCUSSION TITLE
Nginx 502 error after upgrading NETBOX

### DISCUSSION THREAD

**Discussion by uc33260** [netbox-community/netbox#9659] [Q&A]
Hello,

I upgraded my Netbox from 2.11.2 to the latest release 3.5.2.

After the upgrade, when I try to browse the GUI, I have an NGINX 502 error.

# systemctl status netbox.service  command show an error with gunicorn:

● netbox.service - NetBox WSGI Service
     Loaded: loaded (/etc/systemd/system/netbox.service; enabled; vendor preset: enabled)
     Active: activating (auto-restart) (Result: exit-code) since Mon 2022-07-04 13:53:12 UTC; 27s ago
       Docs: https://netbox.readthedocs.io/en/stable/
    Process: 297422 ExecStart=/opt/netbox/venv/bin/gunicorn --pid /var/tmp/netbox.pid --pythonpath /opt/netbox/netbox --config /opt/netbox/gunicorn.py net>
   Main PID: 297422 (code=exited, status=203/EXEC)

Jul 04 13:53:12 netbox systemd[1]: netbox.service: Main process exited, code=exited, status=203/EXEC
Jul 04 13:53:12 netbox systemd[1]: netbox.service: Failed with result 'exit-code'.
lines 1-9/9 (END)

an idea ?

Thank you

**Comment by candlerb** [comment]
Search for [systemd 203](https://github.com/netbox-community/netbox/discussions?discussions_q=systemd+203) in the list archives, or even in Google in general, and you'll see that return code 203 simply means that systemd cannot start the program you've asked it to start.

Typically the executable doesn't exist (you've pointed it at the wrong path) or it doesn't have execute permissions or is not readable by the "netbox" user. On Red Hat / CentOS systems it may be problems with SELinux.

There are some clues if you follow the search link above.
